def example_exception():
    a = 1
    b = 0

    res = a / b
    return res


example_exception()
